index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Peter Okorafor - Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="topnav">
+            <a href="#home" class="logo">Peter Okorafor</a>
+        </div>
+    </header>
+
+    <div class="container">        
+        <p class="container headline">Hey There.<br>I'm Peter</p>
+        <p class="container job-title">I am a Software Developer</p>
+        <p class="container job-description">
+                I can help you build a product, feature or website.
+                Look through some of my work and experience!
+                If you like what you see and have a project you need coded, do not hestiate to contact me.
+        </p>
+    </div>
+
+
+
+    <div class="links">
```

Comment:
Don't forget that it's also considered good practice to use `<ul>` tags for lists, the links and `<a>` tags are fine, but you could try to contain them in different `<li>`'s to make sure your project looks professional :briefcase: